io/net/ethtool/ethtool_tests.sh

```diff
@@ -64,3 +64,5 @@ for command in ifdown ifup; do
         exit $error
     fi
 done
+ifup $interface
+sleep 5
```

Comment:
Something like this would do
```
for i in {1..10};
do
    ip link show ${interface} | grep "state UP"
    if [[ $? == 0 ]]; then
        break;
    fi
done
```